package.json

```diff
@@ -31,7 +31,7 @@
     "mocha": "5.2.0",
     "pdf-to-text": "0.0.6",
     "request": "2.87.0",
-    "request-promise": "4.2.2",
+    "request-promise": "^4.2.2",
```

Comment:
I think for security reasons (don't want packages to suddenly import new code) we want to pin all of our packages to a specific version... is the ^ necessary?